spec/src/main/asciidoc/WebServicesMetadata.adoc

```diff
@@ -127,63 +128,64 @@ This specification addresses the need to simplify:
 * Development of server applications that conform both to basic SOAP and
 WSDL standards.
 * Building Web Services that can be deployed with the core Web Services
-APIs and existing J2EE standards.
+APIs and existing Jakarta EE standards.
 * Separate control of public Web Service message contracts and private
 implementation signatures, because in practice public and private
 formats evolve on different schedules.
 
 It is not a goal of this specification to support every feature or to
 enable the creation of every Web Service that it is possible to write
-within existing specifications (JAX-WS 2.0 [5] provides finer control
+within existing specifications (Jakarta XML Web Services 2.3 [5] provides finer control
 over the resultant Web Service). The goal is to make it easy to build
 the most common types of Web Services.
 
-== 2 Concepts
+== Concepts
 
 This section summarizes the following concepts and processes of the
-JSR-181 specification:
+Jakarta Web Services Metadata specification:
 
-* Programming model for JSR-181 Web Services
-* Use of metadata in JSR-181
+* Programming model for Jakarta Web Services Metadata
```

Comment:
I'd probably use `Programming model for Jakarta Web Services Metadata annotated Web Services`

Reply:
ok, resolved by https://github.com/eclipse-ee4j/jws-api/pull/69/commits/62fdf397f28a148826367da36bcaef2c30e97c7d